### PR TITLE
Fix tilesize

### PR DESCRIPTION
**Pull request by ThierryBerger**
For context: https://discord.com/channels/844211600009199626/1073244711973834822/1073244872775049277

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -265,4 +268,18 @@ mod tests {
...
     }
+
+    #[test]
+    fn query_points_tilesize_10() {
+        let mut db = SparseGrid2d::<10>::default();
+        let e1 = Entity::from_raw(1);
+        let e2 = Entity::from_raw(2);
+        db.insert_point(vec2(12f32, 15f32), e1);
+        db.insert_point(vec2(15f32, 12f32), e2);
+        dbg!("yo");
```
Comment (johanhelsing) [inline src/lib.rs]: Heyo!

File: src/lib.rs [hunk 2]
```diff
@@ -19,8 +19,11 @@ pub struct Aabb {
 
 type Key = (i32, i32);
 
-fn key_from_point(point: Vec2) -> Key {
-    (point.x as i32, point.y as i32)
+fn key_from_point<const TILE_SIZE: usize>(point: Vec2) -> Key {
+    dbg!((
```
Comment (johanhelsing) [inline src/lib.rs]: Didn't know dbg! Returned something. TiIL. Probably still don't need it though :P

**Review by johanhelsing — CHANGES_REQUESTED** [review]
Thanks!

File: src/lib.rs [hunk 3]
```diff
@@ -19,8 +19,11 @@ pub struct Aabb {
 
 type Key = (i32, i32);
 
-fn key_from_point(point: Vec2) -> Key {
-    (point.x as i32, point.y as i32)
+fn key_from_point<const TILE_SIZE: usize>(point: Vec2) -> Key {
```
Comment (ThierryBerger) [inline src/lib.rs:22]: I hesitated to put this method inside `SparseGrid2d` ; you be the judge

Comment (johanhelsing) [reply src/lib.rs:22]: Inside is better 👍

**Comment by johanhelsing** [comment]
Merging now to fix the bug, in case people start using this. We can always move code around later :+1:

**Pull request merged**